packages/sdk/src/contracts/IDelegationRegistry.ts

```diff
@@ -0,0 +1,867 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
I have a bit of a preference to keep all autogenerated files out of our repo, but happy to defer to your intuition here since you are most familiar with the code in this package.

Reply:
++ Would prefer to generate at build time so we always know it's in sync with the relevant abis

Reply:
Sounds good, removing 👍